src/common/low_precision_transformations/CMakeLists.txt

```diff
@@ -51,7 +52,8 @@ target_link_libraries(${TARGET_NAME} INTERFACE openvino::runtime)
 target_include_directories(${TARGET_NAME} INTERFACE
     $<BUILD_INTERFACE:${CMAKE_CURRENT_SOURCE_DIR}/include>
     $<BUILD_INTERFACE:$<TARGET_PROPERTY:ngraph,INTERFACE_INCLUDE_DIRECTORIES>>
-    $<BUILD_INTERFACE:$<TARGET_PROPERTY:inference_engine_transformations,INTERFACE_INCLUDE_DIRECTORIES>>)
+    $<BUILD_INTERFACE:$<TARGET_PROPERTY:inference_engine_transformations,INTERFACE_INCLUDE_DIRECTORIES>>
+    ${CMAKE_CURRENT_SOURCE_DIR}/../..)
```

Comment:
why do we need it?

Reply:
caused by we need include this header "core/src/itt.hpp"

Reply:
each module should have its own itt.hpp
you must not include headers from other module's `src` folder